src/main/java/org/worshipsongs/importer/IAuthorDao.java

```diff
@@ -0,0 +1,15 @@
+package org.worshipsongs.importer;
+
+import java.sql.Connection;
+
+/**
+ * Created by pitchumani on 10/30/15.
+ */
+public interface IAuthorDao
+{
+    Author getAuthor(Author author);
```

Comment:
Consider renaming this method as "findByDisplayName". Change the method parameter to "displayName".